lua/nvim_lsp/racket.lua

```diff
@@ -0,0 +1,23 @@
+local configs = require 'nvim_lsp/configs'
+local util = require 'nvim_lsp/util'
+
+configs.racket = {
+  default_config = {
+    cmd = {"racket", "-l", "racket-langserver"};
+    filetypes = {"racket"};
```

Comment:
Btw, this should probably be `scheme`. Be default `.rkt` files will register as that.

Reply:
You can have both